packages/rari/src/runtime/rsc-client-runtime.ts

```
@@ -787,7 +793,7 @@ class RscClient {
           if (props && props.children) {
             const updatedChildren = renderWithBoundaryUpdates(props.children)
             if (updatedChildren !== props.children) {
-              return cloneElement(element, { ...props, children: updatedChildren } as any)
+              return createElement(element.type, { ...props, children: updatedChildren })
```

Biome (2.4.6)

[error] Avoid passing children using a prop

(lint/correctness/noChildrenProp)